### ISSUE TITLE
Upgrade and update Jekyll layouts, gems and themes

### ISSUE THREAD

**Issue by sondr3** [sondr3/generator-jekyllized#166]
- [ ] Figure out how the new themeing works now that the default theme is it's own gem
- [x] Update the gems that was installed in the previous versions
- [ ] Figure out a better/alternate way of injecting CSS and JS into pages because the current way is really error prone and super brittle. Nor is it very reliable, I don't like it one bit.

**Comment by ghost** [comment]
From what I understand, a lot of new theme directories and files are stored in the gem.  To overwrite them, you move them to the root of the project or make new copies in the root of the project.  

So I think if we download a new version and include all the dirs and files in the jekyll generator we should be good to go.  Is this the way you updated it yesterday?

**Comment by sondr3** [comment]
Yeah, I downloaded the latest version of minima and copied the files into the `app`, I did do some changes to the paths of the SCSS and such, and that’s so that we can integrate build tools properly. Otherwise it’s a straight default install of Jekyll.

I think the next step is getting the prompts back again to fill out `_config.yml` with site name, author information and such. In the last version of this generator I also had to have two separate config files for Jekyll, we should look into if we could just use one single config file instead. We also need to figure out a good way to prompt for social media, where the user can select which services (if any) they want to fill and add to the footer. Other than that I’m looking at webpack right now, I added some comments in #168 about it.

**Comment by ghost** [comment]
In regards to the first check box, layout directories and files can be moved to the root of the project to override gem theme or you can create new _layouts, etc. at the project root.  Jekyll looks at project root first and then to the gem.  Obviously naming conventions are important, but it should be as simple as that!

**Comment by sondr3** [comment]
I saw that in the documentation, my only thought was that for a lot of people they will download this generator so that they can create their own CSS and such with Jekyll and therefore including the whole theme instead of just overriding a few of them would work best. Mostly because it would allow people to quickly iterate with changing the layout instead of having to go to Github and check out how minima does it.

**Comment by ghost** [comment]
I'll look into prompts for the _config.yml.
